content/blog/2023/07/10/2023-07-10-jenkins-june-newsletter.adoc

```diff
@@ -0,0 +1,161 @@
+*** CentOS 7 no longer supported in Jenkins after Nov 16, 2023
+**** Nov 16, 2023 is the likely date for 2.425.1
+* Work on agent and controller images
+** Spread the use of link:https://www.updatecli.io/[updatecli] for the images
+*** ssh-agent is now tracking JDK17
+*** Controller is now tracking Alpine Linux and JDK version for Debian
+** Alpine images now use link:https://docs.oracle.com/en/java/javase/11/tools/jlink.html#GUID-CECAC52B-CFEE-46CB-8166-F17A8E9280E9[jlink] and do not inherit directly from Temurin’s image anymore
+* Latest updates on the docker images:
+** Ssh-agent release link:https://github.com/jenkinsci/docker-ssh-agent/releases/tag/5.6.0[5.6.0] with breaking changes in https://github.com/jenkinsci/docker-ssh-agent/releases/tag/5.4.0[5.4.0]
+** Now tracks the JDK versions.
+*** Bump debian from link:https://hub.docker.com/layers/library/debian/bullseye-20230522/images/sha256-1bf0e24813ee8306c3fba1fe074793eb91c15ee580b61fff7f3f41662bc0031d?context=explore to link:https://hub.docker.com/layers/library/debian/bullseye-20230612/images/sha256-1e8185ca8683c04e6c23cc7bb21b564d54faeb7e20852a6068c85cbdd7bac531?context=explore[bullseye-20230612]
+*** Pins Alpine version to link:https://github.com/jenkinsci/docker-ssh-agent/pull/252[3.17.3]
+*** link:https://github.com/jenkinsci/docker-ssh-agent/pull/251[Tracks] alpine linux base version
+*** Bump Alpine Linux Version to link:https://github.com/jenkinsci/docker-ssh-agent/pull/258[3.18.2]
+*** Bump Git version on Windows to 2.41.0.windows.1
+** Docker-agent release link:https://github.com/jenkinsci/docker-agent/releases/tag/3131.vf2b_b_798b_ce99-2[3131.vf2b_b_798b_ce99-2]
+*** link:https://github.com/jenkinsci/docker-agent/pull/437[Bump] debian from bullseye-20230502 to bullseye-20230612
+*** link:https://github.com/jenkinsci/docker-agent/pull/439[Bump] archlinux from base-20230430.0.146624 to base-20230611.0.157136
+*** link:https://github.com/jenkinsci/docker-agent/pull/434[Bump] Alpine Linux Version to 3.18.2
+*** link:https://github.com/jenkinsci/docker-agent/pull/435[Bump] the Jenkins remoting version to 3131.vf2b_b_798b_ce99
+** Docker inbound agent: link:https://github.com/jenkinsci/docker-inbound-agent/releases/tag/3131.vf2b_b_798b_ce99-2[3131.vf2b_b_798b_ce99-2]
+*** link:https://github.com/jenkinsci/docker-inbound-agent/pull/359[Bump] the parent image jenkins/agent version to 3131.vf2b_b_798b_ce99-2
+** Docker controller link:https://github.com/jenkinsci/docker/releases/tag/2.411[2.411]
+*** Update to Jenkins 2.411
+*** Add link:https://github.com/jenkinsci/docker/pull/1647[more details] to the security policy
+*** link:https://github.com/jenkinsci/docker/pull/1655[Pins] JDK versions for Debian
+*** Windows: link:https://github.com/jenkinsci/docker/pull/1656[update] default jenkins version to 2.410
+*** link:https://github.com/jenkinsci/docker/pull/1653[Pins] Alpine version to 3.18.0
+*** Bump debian from bullseye-20230522 to bullseye-20230612
```

Comment:
I would trim this list to eventually just enumerate the list of dependencies now tracked, or even just their count, no need for all these PRs links in this newsletter IMO.